modules/uncertainty/R/ensemble.R

```diff
@@ -222,6 +222,7 @@ write.ensemble.configs <- function(defaults, ensemble.samples, settings, model,
   }
   
   # See if we need to write to DB
+  if (!is.null(settings$database$bety$write)) {
   write.to.db <- as.logical(settings$database$bety$write)
```

Comment:
Should have the same effect but make it a bit clearer what's happening (I think the fact the existing function completely ignored the `write.to.db` in the function args was a bug).
Might also be worth updating the Roxygen entry on line 198 to something like `@param write.to.db logical: Record this run in BETY? Note that if settings$database$bety$write is set, it will override this`
```suggestion
  if (!is.null(settings$database$bety$write)) {
    # specifying `write` in settings overrides write.to.db in fn args
    write.to.db <- as.logical(settings$database$bety$write)
  }
```

Reply:
I agree. It will help understand the parameter priority.